tools/peptideshaker/ident_params.xml

```diff
@@ -0,0 +1,1259 @@
+                            <option value="1" >High-res LTQ</option>
+                            <option value="2" >TOF</option>
+                            <option value="3" >Q-Exactive</option>
+                        </param>
+                        <param name="msgf_fragmentation" label="MSGF: Fragmentation type" type="select" help="Fragmentation method identifier (used to determine the scoring model)">
+                            <option value="0" selected="True">As written in the spectrum or CID if no info</option>
+                            <option value="1" >CID</option>
+                            <option value="2" >ETD</option>
+                            <option value="3" >HCD</option>
+                        </param>
+                        <param name="msgf_protocol" label="MSGF: Protocol type" type="select" help="Protocol identifier. Protocols are used to enable scoring parameters for enriched and/or labeled samples">
+                            <option value="0" selected="True">Automatic</option>
+                            <option value="1" >Phosphorylation</option>
+                            <option value="2" >iTRAQ</option>
+                            <option value="3" >iTRAQPhospho</option>
+                            <option value="4" >TMT</option>
+                            <option value="5" >Standard</option>
+                        </param>
+                        <param name="msgf_num_matches" label="MSGF: Maximum Number of Spectrum Matches" type="integer" value="1" help="Number of peptide matches per spectrum to report" />
+                        <param name="msgf_additional" label="MS-GF+ additional features" type="select" help="Additional features to export">
+                            <option value="0" selected="True">output basic scores only</option>
+                            <option value="1" >output additional features</option>
+                        </param>
+                    </when>
+                </conditional>
+
+                  <!-- MS-AMANDA ADVANCED PARAMETERS  -->
+                <conditional name="ms_amanda">
+                    <param name="ms_amanda_advanced" type="select" label="MS Amanda Options">
+                        <option value="yes">Advanced</option>
+                        <option value="no" selected="True">Default</option>
+                    </param>
+                    <when value="no" />
+                    <when value="yes">
+                        <param name="ms_amanda_decoy"  type="boolean" truevalue="1" falsevalue="0" checked="false"
+                            label="MS Amanda: Generate Decoys" help="generate decoys" />
+                        <param name="ms_amanda_instrument" label="MS Amanda: instrument" type="select"
+                          help="MS Amanda instrument id option. Available ion types are listed here.">
+
+                          <option value="b, y" selected="True">b, y</option>
+             
```

Comment:
This is not going to work on a cluster system with multiple object stores. You can not rely on the FASTA file is always next to the PAR file :(
In SG and PS you need to specify also the FASTA file, then link everything in `./` and use it.

Reply:
Hi Bjorn, you`ve commented many things I will try to manage step by step.
About this point, I`m not totally sure about what you mean here (and, in fact, the Identification Parameters tool is a key step we need to work perfectly in the workflow). Is your comment doing reference to the tests or to the tool itself?
Talking about the tool itself, I think there should not be any problem. The PAR file is created by the Identification Parameters tool doing reference to the URL of a FASTA file present in the Galaxy history, so it should be present and available using this URL, am I wrong?
The PAR files I included in test-data are different, sure. I didn`t find a different way to test the tool than copying the FASTA to the ./ folder in order to find it.

Reply:
Its about the tool itself.
> Talking about the tool itself, I think there should not be any problem. The PAR file is created by the Identification Parameters tool doing reference to the URL of a FASTA file present in the Galaxy history, so it should be present and available using this URL, am I wrong?

So its not a relative path but on absolute path? This will work in more places, but still not in all. I can imaging cases where job staging is involved where this can break.

In general on of the biggest problem with Proteomic files is that they are referencing some `paths` which can change in various way in HPC/Cloud environments.

Another concern is that people share the param file (to other users), but delete the history or the FASTA file and then PS/SG will not work anymore. This is really counter-intuitive I guess.

Reply:
I`ve talked with Harald about this and we think we can solve this problem if we change the output of the Identification Parameters tool to generate a zip file that includes both the parameters and the FASTA file, in a similar way than PeptideShaker uses a zip file with the output of SearchGUI (with the FASTA, spectrum and parameters files). That way this zip file can be used as input to SearchGUI and the problem of the path is avoided in the same was as the zip file input to PS?

Reply:
Sounds good!